modules/core/test/v2/unit/ethWallet.ts

```diff
@@ -265,6 +265,220 @@ describe('Ethereum Hop Transactions', co(function *() {
   }));
 }));
 
+describe('Ethereum batch transaction data validation', function() {
+  const bitgo = new TestV2BitGo({ env: 'test' });
+  bitgo.initializeTestVars();
+  const coin = bitgo.coin('teth');
+
+  const batcherContractAddress = common.Environments[bitgo.env].batcherContractAddress;
+
+  const initialParams = {
+    recipients: [{
+        address: '0x1111111111111111111111111111111111111111',
+        amount: '5'
+      }, {
+        address: '0x2222222222222222222222222222222222222222',
+        amount: '5'
+      }
+    ]
+  };
+
+  const buildBadParams = (badRecipients) => {
+    const badPrebuildParams = {
+      recipients: badRecipients
+    };
+
+    return {
+      txParams: initialParams,
+      txPrebuild: badPrebuildParams
+    };
+  };
+
+  const runBadBatch = co(function *(badRecipients, expectedErrMsg) {
+    const badParams = buildBadParams(badRecipients);
+    let errMsg = '';
+    let threw = false;
+
+    try {
+      yield coin.verifyTransaction(badParams);
+    } catch (err) {
+      threw = true;
+      errMsg = err.message;
+    }
+
+    threw.should.equal(true, 'batch data verification did not throw when it should have');
+    if (expectedErrMsg) {
+      errMsg.should.equal(expectedErrMsg);
+    }
+  });
+
+  const correctData = '0xcf4c58e2' +
```

Comment:
Consider adding to fixtures, along with the other string constants in this file.

Reply:
Added to fixtures.